src/Test/OpenRiaservices.EndToEnd.Wcf.Test/Data/QueryTests.cs

```diff
@@ -2006,9 +2013,15 @@ public async Task TestServerExceptions_QueryOnNonExistentMethod()
             var query = ctxt.CreateQuery<Product>("NonExistentMethod", null, false, true);
             await ValidateQueryException(ctxt, query, ex =>
             {
-                // REVIEW: Assert the error message.
+#if ASPNETCORE
```

Comment:
consider ignoring the test as you did with `TestInvalidMethodName`

Reply:
Yes will do that